filament/backend/src/opengl/OpenGLContext.cpp

```diff
@@ -71,6 +71,7 @@ OpenGLContext::OpenGLContext() noexcept {
         }
     } else if (strstr(renderer, "Intel")) {
         bugs.vao_doesnt_store_element_array_buffer_binding = true;
+        bugs.disable_feedback_loops = true;
```

Comment:
We may want to leave a TODO to re-evaluate once Intel's discrete GPUs hit the market.

Reply:
Done.